app/controllers/ConfirmSubmissionController.scala

```diff
@@ -259,7 +257,7 @@ class ConfirmSubmissionController @Inject()(mandationStatusCheck: MandationStatu
 
     } recover {
       case exception: AuthorisationException =>
-        logger.warn(s"[ConfirmSubmissionController][buildIdentityData]" +
```

Comment:
same here

Reply:
this error is thrown when an internal server error happens during authorisation, so i think it should be kept as errorLog